src/web/mjs/connectors/Manganelos.mjs

```diff
@@ -0,0 +1,19 @@
+import AnyACG from './templates/AnyACG.mjs';
+
+export default class Manganelos extends AnyACG {
+
+    constructor() {
+        super();
+        super.id = 'manganelos';
+        super.label = 'Manganelos (by AnyACG)';
+        this.tags = [ 'manga', 'webtoon', 'english' ];
+        this.url = 'http://manganelos.com';
+
+        this.queryPages = 'div.chapter-content-inner p#arraydata';
+        this.queryMangaTitle = 'ol > li:nth-child(3)';
+        this.queryMangaTitleText = 'a';
+        this.queryChapters = '#examples a';
```

Comment:
Cannot verify if this is correct since i haven't found any manga that is not redirected to http://manganelo.today